public/javascripts/items.js

```diff
@@ -10,25 +10,83 @@ const fetchParams = {
 const params = completed =>
   Object.assign( {}, fetchParams, { body: JSON.stringify({ completed }) })
 
+const params2 = title =>
+  Object.assign( {}, fetchParams, { body: JSON.stringify({ title }) })
+
+const params3 = description =>
+  Object.assign( {}, fetchParams, { body: JSON.stringify({ description }) })
+
 $(document).ready( () => {
-  $( '.title' ).click( event => {
+  $('.updateTitle').keypress( event => {
+    element = $( event.target )
+    const id = element.data( 'id' )
+
+    if( event.charCode === 13 ) {
+      let updatedTitle = element[0].value
+      fetch( `/items/${id}/edit`, params2( updatedTitle ))
+      .then( result => result.json() )
+      .then( json => {
+        if( json.success ) {
+          let span = element.prev()
```

Comment:
This way of selecting the element to hide feels a little brittle to me - is there some other means of consistently identifying the elements we will want to update?